Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -70,14 +75,18 @@ public void Generate()
             _consoleWriter.WriteLine($"Creating Plugin: {PluginName}");
             var templateFileCreationTask = CreateTemplateFilesForSelection();
             var solutionModificationTask = AddProjectsToSolution();
-            var nugetConfigTask = AddInternalNugetServerToNugetConfig();
-            var msbuildSdkTask = AddMsBuildSdkToGlobalJson();
+            var nugetConfigModificationTask = AddInternalNugetServerToNugetConfig();
+            var globalJsonModificationTask = AddMsBuildSdkToGlobalJson();
```

Comment:
The modification to the global.json file needs to happen before adding references to the project file, because the CentralPackageVersions Sdk can't be resolved otherwise.

Reply:
Addressed in: [Define GenerateAsync method](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/c46d5dad62a378d2357985e37f889aa777a68cc0)